src/features/common/Layout/Header/AccountHeader.module.scss

```diff
@@ -0,0 +1,121 @@
+@import './../../../../theme/theme';
+
+.navContainer {
+  padding: 0px 20px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  top: 100px;
+  left: 0;
+  width: 100%;
+}
+.backButton {
+  box-shadow: 1px 1px 4px -2px #707070;
+  height: 30px;
+  width: 30px;
+  background-color: white;
+  border-radius: 50%;
+  &:hover {
+    cursor: pointer;
+  }
+}
+.settingsButton {
+  width: 30px;
+  height: 30px;
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+// new account header
+
+.headerContainer {
+  // position: fixed;
+  // top:0;
+  background: linear-gradient(
+      to top,
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.2),
+      rgba(0, 0, 0, 0),
+      rgba(0, 0, 0, 0)
+    ),
+    url('https://cdn.plant-for-the-planet.org/media/images/app/bg_layer.jpg')
```

Comment:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1022#discussion_r627960285 is still open, but the code got moved here.